GH readme update for Gloo Gateway (#9701)
solo-io/gloo: devel/contributing/documentation.md

# Documentation

> The documentation is built from a private GitHub. Currently, only Solo.io team members can contribute to this repository. 

Reviewer: ```suggestion
The documentation is built from a private GitHub repository. Currently, only Solo.io team members can contribute to this repository. 
```

> To request changes to the documentation or report an issue, follow the steps in [filing issues](/devel/contributing/issues.md)

Reviewer: maybe this?
```suggestion
To request changes to the documentation or report a problem, follow the steps to [file an issue](/devel/contributing/issues.md)
```
